src/lavinmqctl.cr

```diff
@@ -178,6 +181,7 @@ class LavinMQCtl
     when "set_vhost_limits"      then set_vhost_limits
     when "set_permissions"       then set_permissions
     when "definitions"           then definitions
+    when "hash_password"         then hash_password
```

Comment:
Is this added for this PR, needed? I think this should be separate PR if we want it

Reply:
(And you need that password hash to be able to change the default password)

Reply:
You can create a branch that introduce ctl and api endpoint only and then base this branch upon that. Two different PRs as well.

Reply:
Yeah, sure 👍 I'll split it up next week!